### PR TITLE
Added components to home page

### PR DESCRIPTION
**Pull request by dima2139**
image

Added ui components (4 categories, each has 4 example services)
Header is still written manually, will be changed to defaault ant d header
Most important: added universal props for categories

### CODE REVIEW

File: helper/src/components/Category.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+function Category(props: any) {
+    return (
+    <div>
+        <img src={props.props.img} alt="Pic" width="50" height="40"></img>
+        <ul>
```
Comment (ewvwgeniy) [inline helper/src/components/Category.tsx]: юзай сразу компоненты mui тут, убери ul и li

File: helper/src/components/Header.tsx [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
...qO/5YwEeTVK8JNnIEoYKa07rOjr6Kxu06QA3mS3cx0MZWlkoo0w2KQVeoo3zczdFZzLV87TLZxBjRL9Bu7QajMA+4ytN8n2Ncuz30MkhgPVaQXocmZkwmsUIwixfjjpluw94ZdzbtAdic1iBXair0oCsuSoeNUj21GRSGenFiWJTCNw8AvTyyR4nCsqPDhBFT/24t93P8fdlPZe1I5Kr77jLFc+DbGE5A/siySMnec9PG842BNxhIRJVkoLf9Hhr4qUxUY0yxDXy0CFmyDHZcYjDNuhmuQNF1mM65ioXpMCB2wMlnmW21837CbOO9GyB0cBFo2MipbTyvmcAno/E6amoyvX5dcsf/IGlUq1wKhnM8Mf/xYq9WncUYffRVXKWTPMtVuJfAWTBzqRH+c6iRC74FkgWp485n++fWfWsBERGitmv05TFIwRakjsQK1K4ZRWiiGWkSixvym2bDGy6G9thB66SdkVbSX4q+Fj3nPFgotaJuo7/p+CSWbdFK9oY3R6jZraJutNz3X6uP795G7H9/cDk6vXr16vSBqWVjlEMvSYU7HxTb+UauNkvipjt76krbKvrsBmt+pKIlf230AYREv4homi4XqayDxEHPJYRwPwajjeOFpvtpblOqhjwG+BW9GhhPAG2LdTID3GsQr6XQItVdDvM3dEBjGCAZxXO7PAMr5mBo0Pi6XQ5gth6V+5o0g5rwm00G5iCldLr8tFwjplfs1kMl8YRnWYk5/4nhiz4u3ehBFPI8aEP8czWdA48lSffdO4gmZDbyRXiLJ+6E1rPE6qxxYdBkPljjZcaxuIIvRHJB94HnIXi6W2z1zBU9ZQhwP1BCWNb7bsC+9YcO+GCqDxjGWA5x3vESxLxTtbEAHQ7CWw8V4gla12KrJ3LFPpgsLTWY5X2h2xJ0t6InVsE+tZU29+GSysChaz0SxI7lBpnhDbFnLcvgXspPY224uAxPtgRPEw6kl3xYz6r2bo0XP/p7PFypu4JDeeQD1Yo4mQfCyKTqlp1wb4qlVD2prMVRjUX9ly3lYYxegwxs0O40WVW/U07XNFU32SEAFRGhuabNNQ0dJ/OqTx169evXq1atXr169evXq1atXr169/q3+C2TBJ9reW8Y9AAAAAElFTkSuQmCC" alt="Girl in a jacket" width="50" height="50"></img>
```
Comment (ewvwgeniy) [inline helper/src/components/Header.tsx]: каринку выноси сразу в папку assets

File: helper/src/components/Category.tsx (hunk 1 above)
Comment (dima2139) [reply helper/src/components/Category.tsx]: Done

File: helper/src/components/Header.tsx (hunk 2 above)
Comment (dima2139) [reply helper/src/components/Header.tsx]: Done

**Review by dima2139 — COMMENTED** [review]
image

**Pull request merged**